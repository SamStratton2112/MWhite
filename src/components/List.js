const List = ({list})=>{
    let listItems = list.map(item=><li>{item}</li>);
    return(
        <ul className="leftText scrollText" style={{backgroundColor:'white', borderRadius:'.5em'}}>
            {listItems}
        </ul>
    );
}
export default List;