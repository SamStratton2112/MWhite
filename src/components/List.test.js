import List from './List';
import { render } from '@testing-library/react';

const testList = ['test item 1', 'test item 2']

describe(List, () => {
    it("renders without crashing", () => {
        render(
            <List list={testList} />);
    });
    it("matches snapshot", () => {
        const { asFragment } = render(
            <List list={testList} />);
        expect(asFragment()).toMatchSnapshot();
    });
});