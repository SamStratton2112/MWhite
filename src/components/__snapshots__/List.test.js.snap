// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`List matches snapshot 1`] = `
<DocumentFragment>
  <ul
    class="leftText scrollText"
    style="border-radius: .5em; color: black;"
  >
    <li>
      test item 1
    </li>
    <li>
      test item 2
    </li>
  </ul>
</DocumentFragment>
`;
